source/frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -261,28 +261,20 @@ export const toCqlFilter = (
  * NOTE: The resulting string value is left as-is. It will not be URL-encoded
  * @param object an object to convert to a cql filter string.
  */
-export const toCqlFilterValue = (
-  object: Record<string, string>,
-  forceSimplePid?: boolean,
-  forceExactMatch?: boolean,
-) => {
+export const toCqlFilterValue = (object: Record<string, string>, flags?: IWfsCqlFlags) => {
   const cql: string[] = [];
   Object.keys(object).forEach((key: string) => {
     if (object[key]) {
       if (
         ((key === 'PID' || key === 'PID_PADDED') && object[key]?.length === 9) ||
-        forceExactMatch
+        flags?.forceExactMatch
       ) {
         cql.push(`${key} = '${object[key]}'`);
-      } else if ((key === 'PID' || key === 'PID_PADDED') && object[key] && !forceSimplePid) {
-        cql.push(
-          `PIN ilike '%${object[key]}%' OR PID ilike '%${object[key]}%' OR PID_PADDED ilike '%${object[key]}%'`,
-        );
       } else {
         cql.push(`${key} ilike '%${object[key]}%'`);
       }
     }
   });
 
-  return cql.length ? (forceExactMatch ? cql.join(' OR ') : cql.join(' AND ')) : '';
+  return cql.length ? (flags?.useCqlOr ? cql.join(' OR ') : cql.join(' AND ')) : '';
```

Comment:
allow the or or and of cql concatenation to be determined independently of the forceExactMatch which is also used to control whether ilike or = is used.